spec/models/task_spec.rb

```diff
@@ -24,4 +24,67 @@
       end
     end
   end
+
+  describe "Methods" do
+    before { mock_successful_api_responses(urn: any_args) }
+
+    describe "actions_count" do
+      let!(:task) { create(:task) }
+
+      before { FactoryBot.create_list(:action, 3, task:) }
+
+      it "returns the number of actions associated with the task" do
+        expect(task.actions_count).to eq 3
+      end
+    end
+
+    describe "completed_actions_count" do
```

Comment:
These can be `#completed_action_count` to match the Ruby style of documentation

https://www.betterspecs.org/#describe